content/kubernetes/active-active/preview/prepare-clusters.md

```diff
@@ -34,7 +34,7 @@ For example, if you have two K8s clusters, each with their own REC named `rec1`
 
```

Comment:
at line 19:
"""
This feature is currently in preview and is not for production use.
"""
spoke with Brad and we agree on changing the wording here a bit since it is in public preview customers can use in production but we want them to contact us in case they are interested.
cc: @bascar, @yuvallevy2